test/unit/marked-spec.js

```diff
@@ -1,11 +1,5 @@
 var marked = require('../../lib/marked.js');
 
-it('should run the test', function () {
-  spyOn(marked, 'parse').and.callThrough();
-  marked.parse('Hello World!');
-  expect(marked.parse).toHaveBeenCalled();
-});
-
```

Comment:
I removed this test as well since it didn't really do anything. and was just there initially to have a test